### PR TITLE
Use unused trytes when testing store

### PR DESCRIPTION
**Pull request by thibault-martinez**
Do not merge it yet please.

### CODE REVIEW

File: test/source/api/core_test.cpp [hunk 1]
```diff
@@ -374,39 +362,28 @@ TEST(Core, StoreTransactionsEmpty) {
 }
 
 TEST(Core, StoreTransactionsInvalidTrytes) {
-  IOTA::API::Core api(get_proxy_host(), get_proxy_port());
```
Comment (Cylix) [inline test/source/api/core_test.cpp]: any reason to remove this test?

File: test/source/api/extended/broadcast_and_store_test.cpp [hunk 2]
```diff
@@ -49,30 +49,19 @@ TEST(Extended, BroadcastAndStoreTransactionsInvalidTrytes) {
   EXPECT_GE(res.getDuration(), 0);
 }
 
-TEST(Extended, BroadcastAndStoreTransactionsAlmostInvalidTrytes) {
```
Comment (Cylix) [inline test/source/api/extended/broadcast_and_store_test.cpp]: Same question here

File: test/testnet/scripts/gen_constants.py [hunk 3]
```diff
@@ -243,6 +245,258 @@ def gen_account_bundles_list(account_idx, seed):
     "9999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999"
     "9999999999999999999999999";
 
+//!
+//! Unused trytes to test PoW
+//!
+
+static const std::string UNUSED_TRYTES_1 =
```
Comment (Cylix) [inline test/testnet/scripts/gen_constants.py]: Are these trytes gonna work when regenerating entirely the DB?

File: test/source/api/core_test.cpp (hunk 1 above)
Comment (thibault-martinez) [reply test/source/api/core_test.cpp]: Well, StoreTransactionsInvalidTrytes and StoreTransactionsAlmostInvalidTrytes are the same test.
And StoreTransactionsAlmostInvalidTrytes was actually not tested the way it was intended to.
Both of them were checking api.storeTransactions({ "INVALIDTRYTES" })

File: test/testnet/scripts/gen_constants.py (hunk 3 above)
Comment (thibault-martinez) [reply test/testnet/scripts/gen_constants.py]: Yes, those trytes actually come from the mainnet.
They are just here in order to do pow testing, they do not have to be in the DB.
CI passed so I guess it's OK.

**Pull request merged**